src/app/views/query-response/headers/ResponseHeaders.tsx

```diff
@@ -12,10 +13,19 @@ import { convertVhToPx, getResponseHeight } from '../../common/dimensions-adjust
 
 const ResponseHeaders = () => {
   const { dimensions: { response }, graphResponse, responseAreaExpanded, sampleQuery } = useSelector((state: IRootState) => state);
-  const { headers } = graphResponse;
+  const { body, headers } = graphResponse;
 
   const height = convertVhToPx(getResponseHeight(response.height, responseAreaExpanded), 100);
 
+  const responseIsDownloadUrl = body['contentDownloadUrl'];
+  if (!headers && responseIsDownloadUrl) {
+    return (
+      <MessageBar messageBarType={MessageBarType.warning}>
+        <FormattedMessage id={`Missing response headers for query workaround`}/>
+      </MessageBar>
+    )
+  }
+
```

Comment:
This is nice! It hides the body so that the user doesn't see the internal payload